src/backend/window.rs

```diff
@@ -262,10 +263,10 @@ impl Window {
 		if let Some(image) = &self.image {
 			let image_size = [image.info().width as f32, image.info().height as f32];
 			if !self.options.preserve_aspect_ratio {
-				WindowUniforms::stretch(image_size, [self.zoom; 2], self.pan)
+				WindowUniforms::stretch(image_size)
```

Comment:
It still needs to add the pan and zoom values, right? Would be nice to do that once at the end of the function.

Reply:
Yeah I forgot to commit that